src/components/Heading/Heading.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react'
+import './Heading.css'
 
 const Heading = () => {
-  return <h1 className='text-center mt-5'>Activity Tracker 🌱</h1>
+  return (
+    <div className="heading">
```

Comment:
Did you add a `div` here or was it me? 😄

Reply:
Ilja added this div here. So you should ask him